qiskit_ionq/constants.py

```diff
@@ -63,18 +63,11 @@ class JobStatusMap(enum.Enum):
     FAILED = jobstatus.JobStatus.ERROR.name
 
 
-class AggregationType(enum.Enum):
-    """Class for job results aggregation enumerated type."""
-
-    AVERAGE = "average"
-    PLURALITY = "plurality"
-
-
 class ErrorMitigation(enum.Enum):
     """Class for error mitigation settings enumerated type."""
 
-    SYMMETRIZATION = {"symmetrization": True}
-    NO_SYMMETRIZATION = {"symmetrization": False}
+    DEBIASING = {"debias": True}
+    NO_DEBIASING = {"debias": False}
```

Comment:
I don't like that this requires SDK-side updates if we offer new error mitigation settings.

Is it possible to pass in an arbitrary JSON object for error_mitigation ? Can we support that?

Reply:
im fine with that but then we need to have a source of clear documentation of error mitigation settings that we support - that could be https://docs.ionq.com/#tag/jobs under `error_mitigation`. ill add that as well to the readme

Reply:
Ping me on the docs change if you don't mind; mostly curious on the process for rolling those out